server/copy.go

```diff
@@ -97,7 +101,7 @@ func copyFromFileStore(dst, src string, b filestore.FileBackend) error {
 		if err != nil {
 			return err
 		}
-		f, err := os.Create(fileDest)
+		f, err := os.OpenFile(fileDest, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0660)
```

Comment:
I think this change has no effect as `os.Create(fileDest)` already sets 0660 as permissions. What we had discussed is that only the mattermost process should need to have access and we can restrict the group permissions

Reply:
Ah sorry I forgot to change mode 🤦